peer-feedback.md

```diff
@@ -0,0 +1,27 @@
+# Peer Feedback
+
+We have a developing system of peer feedback to set self-development goals and receive feedback over the course of the year. Around Quarterly Planning meetings, Semi-Annual Retreats, and the Members meetings we have checkins.
+
+## Pair / pod folks for feedback (have a random distribution below)
+
+1. 5 minutes to reflect on work over last year and areas to identify: 
+    - at least 1 thing you did well
+    - 1-3 things you'd like to improve
+2. 15 mins share with your partner
+    - before you start, review this handout on feedback: https://www.seedsforchange.org.uk/feedback.pdf
+    - discuss with your partner!
+    - try to identify how to take action around addressing what you want to improve 
+3. periodic check-in (at quarterly calls)
+    - record these personal feedback and actionable approaches, you will check in with your peer around the next quarterly meeting
+
+Random Distribution Process: roll 7 dice using https://www.random.org/dice/?num=7, work left from right and assign, skip once you have 3 then 2 of a number, re-roll if necessary
```

Comment:
i really like diceware inspo too, but wondering if this perhaps could zero explanation by just copying our initials into https://www.random.org/lists/

But if you prefer dice, I support that aura!

Reply:
adding so folks can pick their poison. I assume this will all get updated.